platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/bfn_extensions/psu_sensors.py

```diff
@@ -0,0 +1,127 @@
+# from netifaces import ifaddresses, AF_INET6
```

Comment:
searching solution for this, since module is not available in pmon
debugging was done in host environment so didn't mention this